Resources/Prototypes/_LostParadise/Datasets/nightmare.yml

```
@@ -0,0 +1,307 @@
+- type: dataset
+  id: nightmare
+  values:
+    #ss13
```

yamllint

[warning] missing starting space in comment

(comments)